src/components/Book.js

```diff
@@ -1,18 +1,23 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Book = ({ id, title, category }) => (
+const Book = ({
+  id, title, category, removeBook, book,
+}) => (
   <tr>
     <td>{id}</td>
     <td>{title}</td>
     <td>{category}</td>
```

Comment:
Also, optionally, instead of passing, `id`, `title`, and `category` separately, you could just pass `book` only and work with that.
```suggestion
const Book = ({
  removeBook, book,
}) => {
  const { id, category, title } = book;
  return (
    <tr>
       <td>{id}</td>
       <td>{title}</td>
       <td>{category}</td>
```

Reply:
These was really a great proposition to implement